ansible/build-gpu-oracle.yml

```diff
@@ -39,4 +39,7 @@
 # docker pull most recent running version of skynet and whisper
     - { role: "gpu-docker-pull", tags: "gpu-docker", gpu_docker_type: "skynet" }
     - { role: "gpu-docker-pull", tags: "gpu-docker", gpu_docker_type: "whisper" }
+    - { role: "streaming-whisper", tags: "gpu-docker", gpu_docker_type: "whisper" }
+    - { role: "autoscaler-sidecar", tags: "autoscaler-sidecar",
+        autoscaler_instance_type: "streaming-whisper", autoscaler_install_flag: true, autoscaler_configure_flag: false, jitsi_autoscaler_sidecar: true}
```

Comment:
This is a shared image for both skynet ande whisper, so we wouldn't want to set the instance type here.